services/wallet/collectibles/collection_data_db_test.go

```diff
@@ -54,11 +57,17 @@ func TestUpdateCollectionsData(t *testing.T) {
 	extraIds := []thirdparty.ContractID{extraID0, extraID1}
 
 	idsNotInDB, err = db.GetIDsNotInDB(extraIds)
+	slices.SortFunc(idsNotInDB, func(a, b thirdparty.ContractID) bool {
```

Comment:
Could use this instead 
![image](https://github.com/status-im/status-go/assets/11161531/8fa5b478-dfa0-487e-86bd-8ed655f3bfc9)